tests/Squidex.Domain.Apps.Entities.Tests/Contents/ContentGrainTests.cs

```diff
@@ -337,7 +365,7 @@ public async Task ChangeStatus_should_create_events_and_update_state_when_archiv
 
             LastEvents
                 .ShouldHaveSameEvents(
-                    CreateContentEvent(new ContentStatusChanged { Status = Status.Archived, Change = StatusChange.Archived })
+                    CreateContentEvent(new ContentStatusChanged { Status = Status.Archived })
```

Comment:
Has Archived been removed as a possible state now? I have noticed it's gone from the ContentChangedTriggerHandlerTests.cs but it's still in here.

Reply:
It is part of the default implementation.